### PR TITLE
People on Discord have problems with deploying to heroku

### PR DESCRIPTION
**Pull request by hdk5**
I asked for opinions on how instructions can be improved and added a few changes.

I think it can be improved even further somehow.

Also, please spellcheck me, because I am not native English speaker.

### CODE REVIEW

**Comment by hdk5** [comment]
Also, I don't think people can update it by merging repos, so maybe updating guide should have easy version "delete old app, create new"?

**Comment by hdk5** [comment]
And there also should be explanation that "web application error" is okay, check logs for salien worker working

**Comment by resir014** [comment]
@hdk5 

> Also, I don't think people can update it by merging repos, so maybe updating guide should have easy version "delete old app, create new"?

Yeah, might be wise to include an easy version of the Heroku guide, since enough people asked for it already.

Also, your guide looks good so far! No spelling errors that I could find.

**Comment by resir014** [comment]
Also keep in mind that I haven't done any deployment in Heroku, but I'll look into going through your setup guide tonight to give some feedback.

Thanks for your work again @hdk5!!! :heart:

**Comment by hdk5** [comment]
Added a few words on update using dropbox, but never tried it myself actually

**Comment by hdk5** [comment]
I am gonna cry. Is all this stuff really such difficult for people?

File: README.md [hunk 1]
```diff
@@ -119,9 +119,13 @@ You can also set up continuous deployment through Docker Hub. [Read the followin
 [![Deploy](https://www.herokucdn.com/deploy/button.svg)](https://heroku.com/deploy)
 
 1. Click the button above.
-2. Set SALIEN_CONFIG_V2 (see note below).
+2. Set SALIEN_CONFIG_V2 ([see note below](#heroku-configuration)).
 3. That's all!
 
+To check if it works, visit logs at https://dashboard.heroku.com/apps/[YOUR_APP_NAME]/logs
+
+"Application error" on web page of your app is fine - it is not related to salien dyno.
```
Comment (resir014) [inline README.md]: Suggested change:

```
If you see "Application Error" when going to the webpage of your app, it's okay - the script will still run anyway.
```

File: README.md [hunk 2]
```diff
@@ -133,29 +137,79 @@ $ git push heroku master
...
 ]
 ```
 
 ### Updating
 
+#### Easy
+
+The easiest way to update script on heroku is to just delete your old app and create new.
+
+Also, you can link your heroku app to dropbox account, download this repository as zip archive and unpack it to selected folder.
```
Comment (resir014) [inline README.md]: Suggested change:

```
You can also link your Heroku app to your Dropbox account. To do that, [download this repository](https://github.com/South-Paw/salien-script-js/archive/master.zip) as a zip archive, and unpack it to the folder created on your Dropbox folder.

For more info on this, visit: https://devcenter.heroku.com/articles/dropbox-sync
```

**Review by resir014 — COMMENTED** [review]
Sorry for the delay, I've given your guide a try just now and it runs with no problems. Thanks a lot for this!

I've got some quick changes that you might consider adding to the readme - nothing big, just some minor wording fixes.

**Review by South-Paw — APPROVED** [review]

**Pull request merged**